### PR TITLE
Improve Catalog registration explanation and docs structure

### PR DESCRIPTION
**Pull request by johannahemminger**
TW-1773; TW-1846; TW-1981; TW-1902

### CODE REVIEW

File: content/en/docs/data-hub/data-hub-catalog/get-started.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## 1 Introduction
+
+Data Hub Catalog is an open, standards-based metadata repository that enables Mendix developers and business admins to discover and explore data sources throughout their connected ecosystem. This respository is accessible to [authorized users](/data-hub/data-hub-catalog/manage-data-sources/user-roles/) in your company on the web app, hub.mendix.com, or through [Data Hub APIs](/apidocs-mxsdk/apidocs/data-hub-apis/).
+
+{{% alert color="info" %}}The Catalog currently supports OData data sources, or services, APIs, or databases that are [wrapped in OData](/refguide/wrap-services-odata/).{{% /alert %}}
+
+## 2 Your Company's Catalog
+
+If you (or your company) have a Mendix license and a Data Hub license, you have a Data Hub Catalog! Log in with your company credentials to hub.mendix.com to see what metadata has been shared in your company.
```
Comment (thijsrohof) [inline content/en/docs/data-hub/data-hub-catalog/get-started.md]: A license is not needed to use the Catalog :) Neither a Data Hub, or a Mendix license.

File: content/en/docs/data-hub/data-hub-catalog/get-started.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+category: "Data Hub Catalog"
+linktitle: "Catalog Quickstart"
+weight: 1
+description: "Describes initial steps beginning users can take with their Catalog."
+tags: ["data hub", "data hub catalog", "consume", "external entities", "asset", "entities", "data hub pane", "studio pro"]
+---
+
+## 1 Introduction
+
+Data Hub Catalog is an open, standards-based metadata repository that enables Mendix developers and business admins to discover and explore data sources throughout their connected ecosystem. This respository is accessible to [authorized users](/data-hub/data-hub-catalog/manage-data-sources/user-roles/) in your company on the web app, hub.mendix.com, or through [Data Hub APIs](/apidocs-mxsdk/apidocs/data-hub-apis/).
```
Comment (thijsrohof) [inline content/en/docs/data-hub/data-hub-catalog/get-started.md]: What are business admins, do we use this terminology in other places?

File: content/en/docs/data-hub/data-hub-catalog/get-started.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+| --- | --- |
+| [Mendix Cloud](/developerportal/deploy/mendix-cloud-deploy/) | Published OData services in a deployed app are [registered automatically](/data-hub/data-hub-catalog/register-data/#mendix-cloud) as data sources in the Catalog <br> [Register sources using the Registration API](/data-hub/data-hub-catalog/register-data/#register-services) |
+| [Private Cloud](/developerportal/deploy/private-cloud/), [Tencent Cloud](/developerportal/deploy/tencent-deploy/) | Using the [Registration API](/data-hub/data-hub-catalog/register-data/#register-services) |
+| [On-Premises](/developerportal/deploy/on-premises-design/) | Using the [Registration API](/data-hub/data-hub-catalog/register-data/#register-services) |
+
+Once data sources are registered, the metadata is stored in the Catalog so that you can share these sources with authorized users in your company.
+
+## 4 Try It Out
+
+Our [Share Data Between Apps](/data-hub/share-data/) guide helps you build an app and use the Catalog to share data. Follow the steps to get a better idea how it works! You can also follow [Write Data to Another App](/data-hub/write-data/) to build on this experience.
```
Comment (thijsrohof) [inline content/en/docs/data-hub/data-hub-catalog/get-started.md]: Maybe be more specific? "use the Catalog to share data between two Mendix apps."

File: content/en/docs/data-hub/data-hub-catalog/manage-data-sources/curate.md [hunk 4]
```diff
@@ -1,20 +1,18 @@
 ---
 title: "Curate Registered Assets"
 url: /data-hub/data-hub-catalog/curate/
-category: "Data Hub Catalog"
-weight: 45
-description: "Describes how to curate and enhance findability of data assets in Data Hub."
+description: "Describes how to curate and enhance findability of data assets in Data Hub Catalog."
```
Comment (thijsrohof) [inline content/en/docs/data-hub/data-hub-catalog/manage-data-sources/curate.md]: maybe also add understandability? So: "escribes how to curate and enhance findability and understandability"

Although, that does sound a bit odd, maybe use another synonym..

File: content/en/docs/data-hub/data-hub-catalog/get-started.md (hunk 1 above)
Comment (johannahemminger) [reply content/en/docs/data-hub/data-hub-catalog/get-started.md]: Wait, then how can someone log into hub.mendix.com if they don't have a Mendix license?

File: content/en/docs/data-hub/data-hub-catalog/get-started.md (hunk 2 above)
Comment (johannahemminger) [reply content/en/docs/data-hub/data-hub-catalog/get-started.md]: No, true, I can adjust the wording here.

File: content/en/docs/data-hub/data-hub-catalog/manage-data-sources/security.md [hunk 5]
```diff
@@ -1,15 +1,13 @@
 ---
 title: "Data Accessibility and Security"
```
Comment (thijsrohof) [inline content/en/docs/data-hub/data-hub-catalog/manage-data-sources/security.md:2]: I think we need to revisit this whole section, as this is highly intertwined with External Entities. This page made some sense when looking at it from a Data Hub perspective, but not so much anymore from a single Catalog perspective imho.

File: content/en/docs/data-hub/data-hub-catalog/manage-data-sources/user-roles.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+## 1 Introduction
+
+Mendix users can use the Data Hub Catalog to search for and consume registered OData services. Users can also register new services and, because they are the data source owners, curate them.
+
+Data Hub Catalog users can do the following: 
+
+* Publish services and register them in the Data Hub Catalog from Studio Pro
+* Register published OData V4 Services for non-Mendix apps manually
```
Comment (thijsrohof) [inline content/en/docs/data-hub/data-hub-catalog/manage-data-sources/user-roles.md]: We support OData v2, 3 & 4 :)

**Review by thijsrohof — CHANGES_REQUESTED** [review]
Hi Johanna, left a few comments and proposed some changes 👍

File: content/en/docs/data-hub/data-hub-catalog/get-started.md (hunk 1 above)
Comment (thijsrohof) [reply content/en/docs/data-hub/data-hub-catalog/get-started.md]: Well, you can just create an account right?

Comment (thijsrohof) [reply content/en/docs/data-hub/data-hub-catalog/get-started.md]: A license (any paid plan) is not needed to use the Catalog.

File: content/en/docs/data-hub/data-hub-catalog/manage-data-sources/security.md (hunk 5 above)
Comment (johannahemminger) [reply content/en/docs/data-hub/data-hub-catalog/manage-data-sources/security.md:2]: I will create a separate Jira issue for this.

**Pull request merged**